docs/resources/cbr_vault.md

```diff
@@ -146,6 +150,30 @@ The following arguments are supported:
 
 * `tags` - (Optional, Map) Specifies the key/value pairs to associate with the CBR vault.
 
+* `charging_mode` - (Optional, String, ForceNew) Specifies the charging mode of the vault.
+  The valid values are as follows:
+  + **prePaid**: the yearly/monthly billing mode.
+  + **postPaid**: the pay-per-use billing mode.
+
+  Changing this will create a new vault.
+
+* `period_unit` - (Optional, String, ForceNew) Specifies the charging period unit of the vault.
+  Valid values are **month** and **year**. This parameter is mandatory if `charging_mode` is set to **prePaid**.
+  Changing this will create a new vault.
+
+* `period` - (Optional, Int, ForceNew) Specifies the charging period of the vault.
+  If `period_unit` is set to **month**, the value ranges from 1 to 9.
+  If `period_unit` is set to **year**, the valid value is 1.
```

Comment:
the `period_unit` for **year** is not correct, please check it.

Reply:
yeah, actually is range from 1 to 5.